src/tsconfig.json

```diff
@@ -9,7 +9,10 @@
         "moduleResolution": "node",
         "noEmit": true,
         "strict": true,
-        "target": "esnext"
+        "target": "esnext",
+        "resolveJsonModule": true,
+        "emitDecoratorMetadata": true,
```

Comment:
uhhhhh this hasn't been implemented into es yet, so why do we need it?

Reply:
I was messing with import and module syntx we can probably remove it